### ISSUE TITLE
Use of aaai15unlabeled sents and spec

### ISSUE THREAD

**Issue by JoshuaMathias** [wjko2/Domain-Agnostic-Sentence-Specificity-Prediction#16]
I've reviewed how the data under aaai15unlabeled is used in the code, but it's still not clear to me and I don't see mention of it in the paper. I'm trying to see if I can renormalize the repository's data to have lower be more specific and higher be less specific, since that's how my data is. I'm also mapping the training data to the rating scheme I'm using.

Questions:

1. Where did these sentences come from, or what was the original source?
2. How do the labels at spec affect training? And why?
3. How were the spec labels generated? I assume they were automated.

**Comment by JoshuaMathias** [comment]
By the way, is the following line of code a carry over from something else?
`dico_label = {'entailment': 0,  'neutral': 1, 'contradiction': 2}`
Entailment and contradiction are different than specificity, so it seems odd that it's mentioned in the code at all at least in terms of labels (not features).

In get_pdtb I've made dico_label dynamic depending on the number of classes.
